packages/environment/environment.js

```diff
@@ -47,15 +47,16 @@ const Environment = {
     const upstreamNetwork = config.network;
     const upstreamConfig = config.networks[upstreamNetwork];
     const forkedNetwork = config.network + "-fork";
-    const ganacheOptions = {
+    const options = {
```

Comment:
I think this variable naming is a little unclear; it's not clear from the names what the distinction is supposed to be between `options` and `ganacheOptions`.  It might have been better to just reassign `ganacheOptions`.

Reply:
Yeah maybe you're right, I'm just slightly allergic to re-assignments. I'll update this :)